static/templates/delete_message_modal_body.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="content">
```

Comment:
What's the need of a `content` class, since it doesn't really change anything and isn't used by any other modal as well.
Not sure on why was this added earlier within the `.html` file as well ?

Also, maybe the file should be renamed to `delete_message_modal` as that's also the case with other modals.